src/model/resolvable.ts

```diff
@@ -18,9 +18,13 @@ import {Feature, ScannedFeature} from './feature';
 /**
  * A ScannedFeature that needs to resolve other Features to build its final
  * representation.
+ *
+ * TODO(aomarks) Is it right that I made this an abstract class? ScannedFeature
```

Comment:
Talked with Justin, cleared up some misunderstandings I had. Going to keep Resolvable an interface in this PR.